vpp-manager/images/init-eks/init_eks.sh

```diff
@@ -88,11 +113,14 @@ configure_machine ()
 		build_and_install_igb_uio
 		sudo insmod /lib/modules/$(uname -r)/kernel/drivers/uio/igb_uio.ko wc_activate=1
 	fi
+
+	# configure hugepages and persist the config across reboots
 	sudo sysctl -w vm.nr_hugepages=${HUGEPAGES}
 	if [ -f /sys/fs/cgroup/hugetlb/kubepods/hugetlb.2MB.limit_in_bytes ]; then
 		echo $((HUGEPAGES * 2 * 1024 * 1024)) | tee /sys/fs/cgroup/hugetlb/kubepods/hugetlb.2MB.limit_in_bytes
 	fi
-	systemctl restart kubelet
+	echo "vm.nr_hugepages=${HUGEPAGES}" >> /etc/sysctl.conf
```

Comment:
We might not want to append everytime we run - but same, in the event we want to keep the script used

Reply:
cloud-init will run only once during first boot thats why this is even required :)

Reply:
of course, there are ways to make cloud-init run every boot but I wouldn't want that